chore(web): update npm version refs for publishing packages 🎡

mcdurdin opened the pull request:
Note that the version entry will be added to the local package.json which will then be dirty. This version entry should not be committed.

@keymanapp-test-bot skip

jahorton commented:
Any chance of verification via a reference `--dry-run` or standalone run of `developer/js/bundle.sh`?  I tried the latter, but...

<img width="599" alt="image" src="https://user-images.githubusercontent.com/25213402/169459421-1cf940f8-8c8c-403f-8052-c6d4c1600694.png">

Note:  `bundle.sh` is not currently set as 'executable' in the repo; I had to fix that to attempt the test.

mcdurdin commented:
> Any chance of verification via a reference --dry-run or standalone run of developer/js/bundle.sh? 

```
$ ./bundle.sh --build-path /tmp/tmp.5dz2i2U9GN
v15.0.233-beta-local
npm notice
npm notice 📦  @keymanapp/lexical-model-compiler@15.0.233-beta-local
npm notice === Tarball Contents ===
npm notice 1.3kB  README.md
npm notice 3.3kB  build.sh
npm notice 4.8kB  bundle.sh
npm notice 4.0kB  libexec/create-override-script-regexp.ts
npm notice 1.6kB  package.json
npm notice 5.5kB  source/errors.ts
npm notice 1.2kB  source/kmlmc.ts
npm notice 3.1kB  source/kmlmi.ts
npm notice 1.5kB  source/kmlmp.ts
npm notice 17.2kB source/lexical-model-compiler/build-trie.ts
npm notice 3.8kB  source/lexical-model-compiler/join-word-breaker-decorator.ts
npm notice 6.7kB  source/lexical-model-compiler/lexical-model-compiler.ts
npm notice 4.9kB  source/lexical-model-compiler/lexical-model.ts
npm notice 4.7kB  source/lexical-model-compiler/model-defaults.ts
npm notice 8.9kB  source/lexical-model-compiler/model-definitions.ts
npm notice 730B   source/lexical-model-compiler/model-info-file.ts
npm notice 2.5kB  source/lexical-model-compiler/script-overrides-decorator.ts
npm notice 5.3kB  source/model-info-compiler/model-info-compiler.ts
npm notice 5.3kB  source/package-compiler/kmp-compiler.ts
npm notice 1.6kB  source/package-compiler/kmp-json-file.ts
npm notice 2.7kB  source/package-compiler/kps-file.ts
npm notice 41B    source/util/min-keyman-version.ts
npm notice 237B   source/util/sysexits.ts
npm notice 2.2kB  source/util/util.ts
npm notice 248B   tests/fixtures/example.qaa.joinwordbreaker/example.qaa.joinwordbreaker.model.ts
npm notice 62B    tests/fixtures/example.qaa.joinwordbreaker/wordlist.tsv
npm notice 267B   tests/fixtures/example.qaa.scriptusesspaces/example.qaa.scriptusesspaces.model.ts
npm notice 130B   tests/fixtures/example.qaa.scriptusesspaces/wordlist.tsv
npm notice 1.0kB  tests/fixtures/example.qaa.sencoten/example.qaa.sencoten.model.kmp.json
npm notice 1.1kB  tests/fixtures/example.qaa.sencoten/example.qaa.sencoten.model.kps
npm notice 146B   tests/fixtures/example.qaa.sencoten/example.qaa.sencoten.model.ts
npm notice 98B    tests/fixtures/example.qaa.sencoten/wordlist.tsv
npm notice 146B   tests/fixtures/example.qaa.smp/example.qaa.smp.model.ts
npm notice 50B    tests/fixtures/example.qaa.smp/wordlist.tsv
npm notice 117B   tests/fixtures/example.qaa.trivial/example.qaa.trivial.model.ts
npm notice 18B    tests/fixtures/example.qaa.trivial/wordlist.tsv
npm notice 119B   tests/fixtures/example.qaa.utf16be/example.qaa.utf16be.model.ts
npm notice 184B   tests/fixtures/example.qaa.utf16be/wordlist.txt
npm notice 119B   tests/fixtures/example.qaa.utf16le/example.qaa.utf16le.model.ts
npm notice 184B   tests/fixtures/example.qaa.utf16le/wordlist.txt
npm notice 212B   tests/fixtures/example.qaa.wordbreaker/example.qaa.wordbreaker.model.ts
npm notice 18B    tests/fixtures/example.qaa.wordbreaker/wordlist.tsv
npm notice 5.2kB  tests/helpers/index.ts
npm notice 209B   tests/README.md
npm notice 10.8kB tests/test-compile-model-with-pseudoclosure.ts
npm notice 1.1kB  tests/test-compile-model.ts
npm notice 6.2kB  tests/test-compile-trie.ts
npm notice 5.2kB  tests/test-default-apply-case.ts
npm notice 7.8kB  tests/test-default-search-term-to-key.ts
npm notice 1.3kB  tests/test-error-logger.ts
npm notice 3.2kB  tests/test-join-word-breaker.ts
npm notice 7.9kB  tests/test-model-definitions.ts
npm notice 1.6kB  tests/test-override-script-defaults.ts
npm notice 1.4kB  tests/test-package-compiler.ts
npm notice 4.8kB  tests/test-parse-wordlist.ts
npm notice 1.3kB  tests/test-punctuation.ts
npm notice 641B   tests/tsconfig.json
npm notice 747B   tsconfig.json
npm notice === Tarball Details ===
npm notice name:          @keymanapp/lexical-model-compiler
npm notice version:       15.0.233-beta-local
npm notice filename:      @keymanapp/lexical-model-compiler-15.0.233-beta-local.tgz
npm notice package size:  42.9 kB
npm notice unpacked size: 156.8 kB
npm notice shasum:        0b67cd16b2c05282731955667a87374b2ac0f6d5
npm notice integrity:     sha512-cWmtn6ed80tqJ[...]h1wnO9/RDmlaQ==
npm notice total files:   58
npm notice 
keymanapp-lexical-model-compiler-15.0.233-beta-local.tgz
package/tests/fixtures/example.qaa.sencoten/example.qaa.sencoten.model.kmp.json
package/package.json
package/tests/tsconfig.json
package/tsconfig.json
package/tests/fixtures/example.qaa.sencoten/example.qaa.sencoten.model.kps
package/README.md
package/tests/README.md
package/build.sh
package/bundle.sh
package/source/lexical-model-compiler/build-trie.ts
package/libexec/create-override-script-regexp.ts
package/source/errors.ts
package/tests/fixtures/example.qaa.joinwordbreaker/example.qaa.joinwordbreaker.model.ts
package/tests/fixtures/example.qaa.scriptusesspaces/example.qaa.scriptusesspaces.model.ts
package/tests/fixtures/example.qaa.sencoten/example.qaa.sencoten.model.ts
package/tests/fixtures/example.qaa.smp/example.qaa.smp.model.ts
package/tests/fixtures/example.qaa.trivial/example.qaa.trivial.model.ts
package/tests/fixtures/example.qaa.utf16be/example.qaa.utf16be.model.ts
package/tests/fixtures/example.qaa.utf16le/example.qaa.utf16le.model.ts
package/tests/fixtures/example.qaa.wordbreaker/example.qaa.wordbreaker.model.ts
package/tests/helpers/index.ts
package/source/lexical-model-compiler/join-word-breaker-decorator.ts
package/source/kmlmc.ts
package/source/kmlmi.ts
package/source/kmlmp.ts
package/source/package-compiler/kmp-compiler.ts
package/source/package-compiler/kmp-json-file.ts
package/source/package-compiler/kps-file.ts
package/source/lexical-model-compiler/lexical-model-compiler.ts
package/source/lexical-model-compiler/lexical-model.ts
package/source/util/min-keyman-version.ts
package/source/lexical-model-compiler/model-defaults.ts
package/source/lexical-model-compiler/model-definitions.ts
package/source/model-info-compiler/model-info-compiler.ts
package/source/lexical-model-compiler/model-info-file.ts
package/source/lexical-model-compiler/script-overrides-decorator.ts
package/source/util/sysexits.ts
package/tests/test-compile-model-with-pseudoclosure.ts
package/tests/test-compile-model.ts
package/tests/test-compile-trie.ts
package/tests/test-default-apply-case.ts
package/tests/test-default-search-term-to-key.ts
package/tests/test-error-logger.ts
package/tests/test-join-word-breaker.ts
package/tests/test-model-definitions.ts
package/tests/test-override-script-defaults.ts
package/tests/test-package-compiler.ts
package/tests/test-parse-wordlist.ts
package/tests/test-punctuation.ts
package/source/util/util.ts
package/tests/fixtures/example.qaa.joinwordbreaker/wordlist.tsv
package/tests/fixtures/example.qaa.scriptusesspaces/wordlist.tsv
package/tests/fixtures/example.qaa.sencoten/wordlist.tsv
package/tests/fixtures/example.qaa.smp/wordlist.tsv
package/tests/fixtures/example.qaa.trivial/wordlist.tsv
package/tests/fixtures/example.qaa.wordbreaker/wordlist.tsv
package/tests/fixtures/example.qaa.utf16be/wordlist.txt
package/tests/fixtures/example.qaa.utf16le/wordlist.txt
v15.0.233-beta-local
npm notice 
npm notice 📦  @keymanapp/models-types@15.0.233-beta-local
npm notice === Tarball Contents ===
npm notice 1.8kB  README.md
npm notice 21.1kB index.d.ts
npm notice 1.0kB  package.json
npm notice === Tarball Details ===
npm notice name:          @keymanapp/models-types
npm notice version:       15.0.233-beta-local
npm notice filename:      @keymanapp/models-types-15.0.233-beta-local.tgz
npm notice package size:  8.2 kB
npm notice unpacked size: 23.9 kB
npm notice shasum:        d00f6ae9f896785c925e135ab6cf88d640b5aa65
npm notice integrity:     sha512-8BKkv6IKxSkmw[...]Cz/gK/YHlyiVw==
npm notice total files:   3
npm notice
keymanapp-models-types-15.0.233-beta-local.tgz

added 22 packages, and audited 23 packages in 9s

found 0 vulnerabilities

up to date, audited 23 packages in 1s

found 0 vulnerabilities
```



> Note: `bundle.sh` is not currently set as 'executable' in the repo; I had to fix that to attempt the test.

Fixed in #6628 (https://github.com/keymanapp/keyman/pull/6628/files#diff-4c59d149a55340806b4db12d19f63e5c71a09295a6c572fc77e11ac731d108c2)
